pisek/opendata/managers.py

```diff
@@ -121,17 +121,29 @@ def __init__(
         super().__init__(f"Check {contestant_output:n}")
 
     def _get_jobs(self) -> list[Job]:
+        jobs: list[Job] = []
+
         self._check = checker_job(
             self._input,
             self._correct_output,
-            self._contestant_output,
+            self._contestant_output.to_sanitized_output(),
             self._test,
             self._seed,
             None,
             self._env,
         )
 
-        return [self._check]
+        sanitize = sanitize_job_direct(
+            self._env,
+            self._contestant_output,
+            self._contestant_output.to_sanitized_output(),
+            False,
+        )
+        if sanitize is not None:
+            jobs.append(sanitize)
+            self._check.add_prerequisite(sanitize, name="sanitize")
+
+        return jobs + [self._check]
```

Comment:
Nit: Isn't it a bit confusing to create the jobs in reverse order? Especially if only one uses the `jobs` list?